### ISSUE TITLE
Grafana SSL wildcard not updating with TRMM

### ISSUE THREAD

**Issue by WolfgangZA** [dinger1986/TRMM-Grafana#8]
Upon first installation everything worked as intended.
Once TRMM Wildcard expired, Documentation for TRMM was followed to update TRMM certificate (traditional install)
TRMM works fine but Grafana's ssl cert is expired, Even if Grafana is updated it doesnt pull the new TRMM SSL cert to Grafana.

Either this step is missing from documentation here or there could be a bug with it.

**Comment by dinger1986** [comment]
You can try: sudo systemctl restart grafana-server

Open */etc/grafana/grafana.ini* in your favourite editor and check what’s listed on this line *cert_file = /etc/letsencrypt/live/${certdomain}/fullchain.pem*

Also if that doesn't help I would like some screenshots of the setup.

**Comment by WolfgangZA** [comment]
Hi Dinger. 

Ok i have managed to resolve this issue.
I have updated TRMM last night to the latest v11.3
Thereafter i updated the server and grafana via sudo apt-get update & upgrade.

I then proceeded with sudo systemctl restart grafana-server and then rebooted ubuntu server as a whole.

Only after that did the new wildcard cert populate under grafana.

Many thanks for the swift response and assistance.
